algorithmic_efficiency/workloads/fastmri/fastmri_jax/workload.py

```diff
@@ -156,3 +162,32 @@ def _eval_model_on_split(self,
           k: v + synced_metrics[k][0] for k, v in total_metrics.items()
       }
     return {k: float(v.item() / num_examples) for k, v in total_metrics.items()}
+
+
+class FastMRIModelSizeWorkload(FastMRIWorkload):
```

Comment:
Are there not the adjusted validation and test targets missing here and for the Jax variants as well?

Reply:
Yes there are but we have to redo the target setting because of a bug we had in the init2winit code. I'll add this note in the PR.